generate-patch-release-notes.sh

```diff
@@ -48,7 +48,7 @@ function gen_patch_relnotes() {
   rm out.md || true
```

Comment:
after I run this script, it doesn't cleanup the out.md.

```
(⎈|kind-kind:N/A)➜  csi-driver-host-path git:(changelog-release-1.12) ✗ git status
On branch changelog-release-1.12
Your branch is ahead of 'upstream/release-1.12' by 1 commit.
  (use "git push" to publish your local commits)

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	CHANGELOG/out.md

nothing added to commit but untracked files present (use "git add" to track)
```

Reply:
fixed